emukit/experimental_design/model_based/experimental_design_loop.py

```diff
@@ -42,3 +42,5 @@ def __init__(self, space: ParameterSpace, model: IModel, acquisition: Acquisitio
         loop_state = create_loop_state(model.X, model.Y)
 
         super().__init__(candidate_point_calculator, model_updater, loop_state)
+
+        self.model = model
```

Comment:
Is there a reason why this is defined after super and the other stuff before?

Reply:
The other stuff is needed to calculate the inputs to super and this does not